packages/md-enhance/src/node/markdown-it/stylize.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * 对<tag>text</tag>形式的token组合(inline)，进行显示增强：
+ * ①对tag增加属性（如class样式）②对text进行替换（如Emoji后缀）。
+ * 其主要目的是使特定词汇具在render后，变得更醒目或生动。
+ * 可以通过front matter设置 noStylize:[t1,t2]禁用t1和t2的增强。
+ * 注意，所有配置项都区分大小写。
+ *
+ * @author trydofor
+ * @since 2022-05-20
+ * @see https://markdown-it.github.io
+ */
+
+import * as Token from "markdown-it/lib/token";
```

Comment:
We would prefer 

```suggestion
import type { default as  Token } from "markdown-it/lib/token";
```

to be constant with other files